hyperion/app/protos/hyperion.proto

```
@@ -1,63 +1,45 @@
 syntax = "proto3";
 
-package de.tum.cit.aet.artemis.hyperion;
+package de.tum.cit.aet.artemis.hyperion.v2;
```

Buf (1.54.0)

Files with package "de.tum.cit.aet.artemis.hyperion.v2" must be within a directory "de/tum/cit/aet/artemis/hyperion/v2" relative to root but were in directory "hyperion/app/protos".

(PACKAGE_DIRECTORY_MATCH)